promptsource/templates/selqa/answer_selection_analysis/templates.yaml

```diff
@@ -18,9 +18,11 @@ templates:
   5354e98d-8aa2-49d0-a50b-fc72a503d7d4: !Template
     answer_choices: No ||| Yes
     id: 5354e98d-8aa2-49d0-a50b-fc72a503d7d4
-    jinja: '{% set possible_indexes = [] %}{% for c in candidates %}{% if c|trim %}{% set _ = possible_indexes.append(loop.index0) %}{% endif %}{% endfor %}{% set rand_index = possible_indexes | choice %} Would it make
-      sense to reply "{{ candidates[rand_index]|trim|trim(''.'') }}" to the question
-      "{{ question }}"? ||| {% if rand_index in answers %}{{answer_choices[1]}}{%else %}{{answer_choices[0]}}{%endif%}'
+    jinja: '{% set rand_index = range(0, candidates|length)|random %}
```

Comment:
Admittedly, the change won't check empty strings anymore. Yet I guess we do not need that?